### ISSUE TITLE
Conflicting results for test 16 "Read information about a menu item" (Action Menu Button Example Using aria-activedescendant)

### ISSUE THREAD

**Issue by IsaDC** [w3c/aria-at#985]
## Description of Behavior

@mcking65  Could you run this test, please?
@jha11y  and I got very different results.

## Test Setup

- Test File: [test-16-read-information-about-menu-item-interaction-voiceover_macos.collected.html](https://aria-at.netlify.app/tests/menu-button-actions-active-descendant/test-16-read-information-about-menu-item-interaction-voiceover_macos.collected.html)
- AT: VoiceOver for macOS (version 13.4.1)
- Browser: Safari (version 16.5.1)

## Review Conflicts for "Read information about a menu item"

1.  ### Assertion Results for "Control+Option+F3" Command and "Name of the focused item, 'Action 1', is conveyed" Assertion
    
    *   Tester jha11y recorded output "menu is in the VoiceOver cursor You are currently in a menu" and marked assertion as failing with no output.
    *   Tester IsaDC recorded output " Run Test Setup dimmed button is in the VoiceOver cursor" and marked assertion as failing with incorrect output.
2.  ### Assertion Results for "Control+Option+F3" Command and "Position of the focused item in the menu (1) is conveyed" Assertion
    
    *   Tester jha11y recorded output "menu is in the VoiceOver cursor You are currently in a menu" and marked assertion as failing with no output.
    *   Tester IsaDC recorded output " Run Test Setup dimmed button is in the VoiceOver cursor" and marked assertion as failing with incorrect output.
3.  ### Assertion Results for "Control+Option+F3" Command and "Number of items in the menu (4) is conveyed" Assertion
    
    *   Tester jha11y recorded output "menu is in the VoiceOver cursor You are currently in a menu" and marked assertion as failing with no output.
    *   Tester IsaDC recorded output " Run Test Setup dimmed button is in the VoiceOver cursor" and marked assertion as failing with incorrect output.
4.  ### Assertion Results for "Control+Option+F4" Command and "Role of the focused item, 'menu item', is conveyed" Assertion
    
    *   Tester jha11y recorded output "menu has keyboard focus You are currently in a menu" and marked assertion as failing with incorrect output.
    *   Tester IsaDC recorded output " Actions menu 4 items Actions menu has keyboard focus" and marked assertion as failing with no output.
5.  ### Assertion Results for "Control+Option+F4" Command and "Number of items in the menu (4) is conveyed" Assertion
    
    *   Tester jha11y recorded output "menu has keyboard focus You are currently in a menu" and marked assertion as failing with no output.
    *   Tester IsaDC recorded output " Actions menu 4 items Actions menu has keyboard focus" and marked assertion as passing.

**Comment by mcking65** [comment]
Resolved by Joe re-running using same conditions as Isabel.
